packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -199,7 +199,7 @@ const useNavbarStateManager = (props: HookProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []); // <-- run this only once!!
 
-  React.useEffect(() => {
+  React.useLayoutEffect(() => {
```

Comment:
The `useLayoutEffect` is just an additional improvement which ensures that the effect is applied to the DOM before the component is rendered.

https://kentcdodds.com/blog/useeffect-vs-uselayouteffect

Reply:
🔥